src/ktn_debug.erl

```diff
@@ -11,7 +17,7 @@
 -spec ppst() ->
   [any()].
 ppst() ->
-  ppst(erlang:get_stacktrace()).
+  ppst(?GET_STACKTRACE).
```

Comment:
Instead of doing this, just remove `ppst/0`. We don't need to be _that_ backwards compatible. When I generate the next release, I'll make sure to bump the _minor_ version.

Reply:
Well, if you're breaking interface, at least according to semver (I'm not assuming you're using it) you'd be bumping the _major_ version.

Reply:
Major version… it will be.

Reply:
f1ddd18